client/context/verifier_test.go

```diff
@@ -0,0 +1,35 @@
+package context_test
+
+import (
+	"io/ioutil"
+	"testing"
+
+	"github.com/cosmos/cosmos-sdk/client/context"
+	"github.com/stretchr/testify/require"
+)
+
+func TestCreateVerifier(t *testing.T) {
+	tmpDir, err := ioutil.TempDir("", "example")
+	require.NoError(t, err)
+
+	testCases := []struct {
+		name      string
+		ctx       context.CLIContext
+		expectErr bool
```

Comment:
Can we have one case with `expectErr = false`?

Reply:
Looking at the Tendermint verifier code, unfortunately, I think that will require a running chain. So I don't think it's possible :-/